lib/widgets/rules_buttons.dart

```diff
@@ -1,7 +1,12 @@
 import 'package:flutter/material.dart';
+import 'package:tic_tac_toe/widgets/dropdown.dart';
 
 import 'game_view.dart';
 
+String selectedValue = '3';
```

Comment:
These globals should go away too.

There should not be global variables at all in the application.
(Global constants ar ok)

Reply:
Moved vars to Widget class